specifyweb/frontend/js_src/lib/components/DataModel/businessRuleUtils.ts

```diff
@@ -7,6 +7,8 @@ export const CURRENT_DETERMINATION_KEY = 'determination-isCurrent';
 export const COG_TOITSELF = 'cog-toItself';
 export const PARENTCOG_KEY = 'cog-parentCog';
 export const COG_PRIMARY_KEY = 'cog-isPrimary';
+export const PRIMARY_RECORD =
+  'This record cannot be deleted as it is the primary record of the Collection Object Group. Please reload the page, then assign another CO as the primary record if a change is desired.';
```

Comment:
This file has blocker keys. Looks like this should go in `deletePrimaryRecord` in `resources.ts` instead?